must-gather/collection-scripts/gather_ceph_resources

```diff
@@ -280,17 +280,14 @@ for ns in $namespaces; do
             isEnabled=$(timeout 60 oc -n openshift-storage exec "${HOSTNAME}"-helper -- bash -c "rbd mirror pool info $bp --format=json | jq --raw-output '.mode'")
             if [ "${isEnabled}" != "disabled" ]; then
                 { printf "Mirroring is enabled on: %s\n" "${bp}" >> "${COMMAND_OUTPUT_FILE}"; }
-                { timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "rbd mirror snapshot schedule list --pool=$bp --fromat=json" >> "${COMMAND_OUTPUT_FILE}"; } >> "${COMMAND_ERR_OUTPUT_DIR}"/gather-rbd-mirror-snap-schedule-list-"${bp}"-debug.log 2>&1 &
-                pids_ceph+=($!)
-                { timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "rbd mirror pool status $bp --format=json" >> "${COMMAND_OUTPUT_FILE}"; } >> "${COMMAND_ERR_OUTPUT_DIR}"/gather-rbd-mirror-pool-status-"${bp}"-debug.log 2>&1 &
+                { timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "rbd mirror pool status --verbose $bp --format=json --pretty-format" >> "${COMMAND_OUTPUT_DIR}"/rbd_mirror_pool_status_"${bp}"; } >> "${COMMAND_ERR_OUTPUT_DIR}"/gather-rbd-mirror-pool-status-"${bp}"-debug.log 2>&1 &
                 pids_ceph+=($!)
-                { timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "rbd mirror pool info $bp --format=json" >> "${COMMAND_OUTPUT_FILE}"; } >> "${COMMAND_ERR_OUTPUT_DIR}"/gather-rbd-mirror-pool-info-"${bp}"-debug.log 2>&1 &
+                { timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "rbd mirror pool info $bp --format=json --pretty-format" >> "${COMMAND_OUTPUT_DIR}"/rbd_mirror_pool_info_"${bp}"; } >> "${COMMAND_ERR_OUTPUT_DIR}"/gather-rbd-mirror-pool-info-"${bp}"-debug.log 2>&1 &
                 pids_ceph+=($!)
                 images=$(timeout 60 oc -n openshift-storage exec "${HOSTNAME}"-helper -- bash -c "rbd ls -p $bp")
                 for image in $images; do
                     { printf "Printing information for image: %s\n" "${image}" >> "${COMMAND_OUTPUT_FILE}"; }
-                    { timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "rbd mirror image status $bp/$image --format=json" >> "${COMMAND_OUTPUT_FILE}"; } >> "${COMMAND_ERR_OUTPUT_DIR}"/gather-rbd-mirror-image-status-"${bp}"-"${image}"-debug.log 2>&1 &
-                    pids_ceph+=($!)
+                    { timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "rbd mirror image status $bp/$image --format=json --pretty-format" >> "${COMMAND_OUTPUT_DIR}"/rbd_mirror_image_status_"${bp}"; } >> "${COMMAND_ERR_OUTPUT_DIR}"/gather-rbd-mirror-image-status-"${bp}"-"${image}"-debug.log 2>&1
```

Comment:
The reason I suggested adding a pool name suffix was to work around these parallelization issues.  Now that there is a good example in https://github.com/red-hat-storage/ocs-operator/pull/1611, it would be great if we could do the same here -- collect all rbd commands for a particular pool and wait for them before moving on to the next pool.  The resulting files wouldn't need pool name suffixes then.

In pseudo-bash code:
```
for each block pool as bp:
    pids_rbd=()
    if mirroring is enabled:
        {
            printf "Collecting mirror pool info for: $bp\n"
            rbd mirror pool info $bp --format=json --pretty-format
        } >> .../rbd_mirror_pool_info &
        pids_rbd+=($!)
        {
            printf "Collecting mirror pool status for: $bp\n"
            rbd mirror pool status --verbose $bp --format=json --pretty-format
        } >> .../rbd_mirror_pool_status &
        pids_rbd+=($!)
        for each image in bp as image:
            {
                printf "Collecting mirror image status for: $bp/$image\n"
                rbd mirror image status $bp/$image --format=json --pretty-format
            } >> .../rbd_mirror_image_status_$image.part &
            pids_rbd+=($!)
        wait for pids_rbd
        merge .../rbd_mirror_image_status_*.part into .../rbd_mirror_image_status
    else:
        printf "Mirroring is disabled on: $bp\n" >> rbd_mirror_pool_info
        printf "Mirroring is disabled on: $bp\n" >> rbd_mirror_pool_status
        printf "Mirroring is disabled on: $bp\n" >> rbd_mirror_image_status
```